src/spikeinterface/core/channelsaggregationrecording.py

```diff
@@ -13,6 +13,8 @@ class ChannelsAggregationRecording(BaseRecording):
 
     """
 
+    _skip_properties = []
+
```

Comment:
This is too dangerous!
Imagine we happen some other propertis in `Baserecording._skip_properties` ?
We will totally forget about this.
I propose to remove this.

Reply:
I think it's ok! For these recordings the noise levels are exactly the same

Reply:
This is not my point. Imagine we have more properties to be protected than theses two one.
What will will do is to put then in RecordingBase and without thinking to check this everywhere.
A _skip_properties reset at some places is dangerous.

Reply:
ok, done